### DISCUSSION TITLE
mongoose-paginate-v2 alternative for prisma

### DISCUSSION THREAD

**Discussion by masterbater** [prisma/prisma#13442] [Q&A] [answered]
Guys can you share me your solution or any libraries you use that can replicate  mongoose-paginate-v2  plugin

I did know that prisma client has built in pagination want to cover all those below 
```
{
data: [ ]
hasNextPage: true
hasPrevPage: false
limit: 10
nextPage: 2
page: 1
pagingCounter: 1
prevPage: null
totalDocs: 5264
totalPages: 527}
```

**Comment by nurul3101** [comment]
Hey @masterbater 👋 
Do you mean that you want all the pagination data that you mentioned in the snippet?
You might be interested in this pagination example: [Cursor based pagination](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#cursor-based-pagination)

**Reply by vladi-stevanovic** [reply]
👋 Hello @masterbater  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment, marked as answer]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
